.editorconfig

```diff
@@ -0,0 +1,261 @@
+# Удалите строку ниже, если вы хотите наследовать параметры .editorconfig из каталогов, расположенных выше в иерархии
```

Comment:
Can you regenerate this file with visual studio in english such that this file has english comments?
If you don't know how i might try …
Thank you

Reply:
Try to open Tools -> Options -> Editor -> C# -> Code Style and press on the button "Generate .editorconfig"

Reply:
It sholld generate same config but with comments on yours language